src/patternfly/components/Tabs/tabs.scss

```diff
@@ -1,106 +1,254 @@
+$pf-c-tabs--breakpoint-map: build-breakpoint-map("base", "md", "lg", "xl", "2xl");
+$pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
+
 .pf-c-tabs {
-  // Primary tab items
-  --pf-c-tabs__item--BackgroundColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-tabs__item--BorderColor: var(--pf-global--BorderColor--100);
-  --pf-c-tabs__item--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-tabs__item--m-current--ZIndex: var(--pf-global--ZIndex--sm);
-  --pf-c-tabs__item--m-current--Color: var(--pf-global--active-color--100);
-  --pf-c-tabs__item--m-current--BorderTopWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-tabs--Inset: 0;
+  --pf-c-tabs--before--BorderStyle: solid;
+  --pf-c-tabs--before--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-tabs--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-tabs--before--BorderTopWidth: 0;
```

Comment:
are these vars that represent 0 necessary?

Reply:
Yeah, the zero value vars are updated based on which context the button is in. It also allows the consumer to set `border-width` s individually or change all `border-width` s uniformly.